libraries/AP_HAL_Linux/RCInput_Aero.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017  Intel Corporation. All rights reserved.
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+/* feature added by Anemos Technologies : Philippe Crochat (pcrochat@anemos-technologies.com) */
+
+#pragma once
+
+#define DEVICE_NAME "aeroio"
+#define CHANNEL_NUMBER 8
+
+#define CHANNEL_MIN_VALUE 970
+#define CHANNEL_MAX_VALUE 2000
```

Comment:
humn... this looks strange. we should respect the parameters, not a hardcoded value like this.

Reply:
It is because the FPGA is not stable enough, we will get rid of those odd values once the FPGA firmware is cleaned